package-lock.json

```diff
@@ -1,8 +1,7644 @@
 {
   "name": "@newrelic/apollo-server-plugin",
   "version": "0.1.3",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Interested in the change here...

What version of npm are you using?

Reply:
``` sh
$ npm -v
7.10.0
```

i will revert it